fe1-web/parts/lao/socialMedia/SocialSearch.tsx

```diff
@@ -1,11 +1,67 @@
 import * as React from 'react';
-import { Text, View } from 'react-native';
+import {
+  FlatList, ListRenderItemInfo, StyleSheet, View, ViewStyle,
+} from 'react-native';
 import STRINGS from 'res/strings';
+import { makeCurrentLao, makeLastRollCallAttendeesList } from 'store';
+import { useSelector } from 'react-redux';
+import { PublicKey } from 'model/objects';
+import TextBlock from 'components/TextBlock';
+import UserListItem from 'components/UserListItem';
+import { gray } from 'styles/colors';
 
-const SocialSearch = () => (
-  <View>
-    <Text>{STRINGS.social_media_navigation_tab_search}</Text>
-  </View>
-);
+/**
+ * Component that will be used to allow users to search for other users or topics.
+ * For now, it is used to show all the attendees of the last roll call so that everyone can follow
+ * whoever they want.
+ */
+const styles = StyleSheet.create({
+  viewCenter: {
+    alignSelf: 'center',
+    width: 600,
+  } as ViewStyle,
+  titleTextView: {
+    alignSelf: 'flex-start',
+    marginTop: 20,
+  } as ViewStyle,
+  attendeesList: {
+    flexDirection: 'column',
+    marginTop: 20,
+    borderTopWidth: 1,
+    borderColor: gray,
+  } as ViewStyle,
+});
+
+const SocialSearch = () => {
+  const laoSelect = makeCurrentLao();
+  const currentLao = useSelector(laoSelect);
+
+  if (!currentLao) {
+    throw new Error('Impossible to open Social media Search if you are not connected to a LAO');
```

Comment:
Out of scope for this PR:
Would it make sense to have a more "robust" behavior to this kind of errors ?

Reply:
I think so, since I need to put it pretty much everywhere since `makeCurrentLao()` is used a lot. What would you suggest ? It can be done in a separated PR.